sorc/grid_tools.fd/regional_esg_grid.fd/pmat5.f90

```diff
@@ -199,77 +198,81 @@ subroutine dctogr(xe,rlat,rlon)!                                       [ctogr]
 endif
 end subroutine dctogr
 
-!=============================================================================
+!> utility routine to calculate
```

Comment:
Utility routine to calculate what?

Reply:
> Utility routine to calculate what?

There is no indication in original source code to your question. @GeorgeGayno-NOAA do you have any suggestion?